docs/anomalies/manage-anomalies.md

```diff
@@ -0,0 +1,422 @@
+![all](../assets/datastores/manage-anomalies/all-dark-9.png#only-dark)
+
+## Acknowledge Anomalies
+
+By acknowledging anomalies, you indicate that they have been reviewed or recognized. This can be done either individually or in bulk, depending on your workflow. Acknowledging anomalies helps you keep track of issues that have been addressed, even if further action is still required.
+
+!!! warning 
+    Once an anomaly is acknowledged, it remains acknowledged and never reverts to the active state.
+
+### Method I. Acknowledge Specific Anomaly
+
+You can acknowledge individual anomalies either directly or through the action menu, giving you precise control over each anomaly's status.
+
+#### 1. Acknowledge Directly
+
+**Step 1:** Locate the active anomaly you want to acknowledge, and click on the acknowledge icon (represented by an **eye** icon) located on the right side of the anomaly. 
+
+![acknowledge-directly](../assets/datastores/manage-anomalies/acknowledge-directly-light-10.png#only-light)
+![acknowledge-directly](../assets/datastores/manage-anomalies/acknowledge-directly-dark-10.png#only-dark)
+
+A modal window titled **“Acknowledge Anomaly”** will appear, confirming that this action acknowledges the anomaly as a legitimate data quality concern.
+
+You also have the option to leave a comment in the provided field to provide additional context or details. 
+
+![acknowledge-anomaly](../assets/datastores/manage-anomalies/acknowledge-anomaly-light-11.png#only-light)
+![acknowledge-anomaly](../assets/datastores/manage-anomalies/acknowledge-anomaly-dark-11.png#only-dark)
+
+**Step 2:** Click on the **Acknowledge** button to move the anomaly to the acknowledged state.
+
+![acknowledge-button](../assets/datastores/manage-anomalies/acknowledge-button-light-12.png#only-light)
+![acknowledge-button](../assets/datastores/manage-anomalies/acknowledge-button-dark-12.png#only-dark)
+
+**Step 3:** After clicking on the **Acknowledge** button your anomaly is successfully moved to the acknowledge and a flash message will appear saying **“The Anomaly has been successfully acknowledged.”**
+
+![acknowledge-msg](../assets/datastores/manage-anomalies/acknowledge-msg-light-13.png#only-light)
+![acknowledge-msg](../assets/datastores/manage-anomalies/acknowledge-msg-dark-13.png#only-dark)
+
+#### 2. Acknowledge via Action Menu
+
+**Step 1**: Click on the active anomaly from the list of available anomalies that you want to acknowledge.
+
+![active-anomaly](../assets/datastores/manage-anomalies/active-anomaly-light-14.png#only-light)
+![active-anomaly](../assets/datastores/manage-anomalies/active-anomaly-dark-14.png#only-dark)
+
+**Step 2:** A modal window will appear displaying the anomaly details. Click on the **vertical ellipsis** **(⋮)** located in the upper-right corner of the modal window, and click on **“Acknowledge”** from the drop-down menu. 
+
+![vertical-acknowledge](../assets/datastores/manage-anomalies/vertical-acknowledge-light-15.png#only-light)
```

Comment:
Replace this image with one that user can see the `Acknowledge` button visible. This screenshot shows that the button is disabled.